autopush/endpoint.py

```diff
@@ -764,6 +982,21 @@ def post(self, uaid=""):
         # matches up
         new_uaid = False
 
+        # normalize the path vars into parameters
+        router_token = router_token or params.get('router_token')
+        router_type = router_type or params.get('type')
+        if router_type not in self.ap_settings.routers:
+            log.msg("Invalid parameters", **self._client_info())
+            return self._write_response(
+                400, 108, message="Invalid arguments")
+        router = self.ap_settings.routers[router_type]
+        (valid, router_token) = router.check_token(router_token)
+        if not valid:
+            newUrl = self._relocate(router_type, router_token, uaid, chid)
+            return self._write_response(
+                301, 0, message=("Location: %s" % newUrl),
+                headers={"Location": newUrl})
```

Comment:
Yes, senderids are not considered private since you need the Auth Token to actually send out the data.